addons/animalTransport/XEH_postClientInit.sqf

```diff
@@ -0,0 +1,70 @@
+#include "script_component.hpp"
+
+LOG("postClientInit");
+
+if (!hasInterface) exitWith {};
+
+{
+    private _interactPoint = ([_x >> "GRAD_AnimalTransport", "actionPoint", [0, 0, 0]] call BIS_fnc_returnConfigEntry);
+
+    private _mainAction = [
+        "ACE_MainActions",
```

Comment:
note: it has to be `ACE_MainActions` as we rely on ACE_dragging to latch onto this and add its action as a child